video/external/src/backend.rs

```diff
@@ -82,6 +82,10 @@ impl ExternalVideoBackend {
                 "libopenh264-2.4.1-mac-arm64.dylib",
                 "213ff93831cfa3dd6d7ad0c3a3403a6ceedf4ac1341e1278b5b869d42fefb496",
             ),
+            ("ios", _) => (
+                "libopenh264-2.4.1-ios.a",
```

Comment:
Isn't this a static library? If so, what good is downloading it at runtime? 🤔

Reply:
You're right, that makes no sense, I didn't think when I migrated that part